Include/memoryobject.h

```diff
@@ -56,6 +56,7 @@ typedef struct {
 
 typedef struct {
     PyObject_VAR_HEAD
+    int readonly;
```

Comment:
The `Py_buffer` structure already a `readonly` field, so you don't need to add this. See https://docs.python.org/3/c-api/buffer.html#buffer-structure